views/includes/entity.html

```diff
@@ -34,6 +34,43 @@
   %}
 {% endif %}
 
+{% set isEmailPrivate %}
+    {% if not appData.entity.email and pageParams.isStatement %}
+      <strong class="govuk-tag govuk-tag--blue">CANNOT BE DISPLAYED</strong>
+    {% elif appData.entity.email %}
+      {{ appData.entity.email }}
+    {% endif %}
+{% endset %}
+
+{% if pageParams.isStatement %}
+  {% set jurisdiction = 
+    {
+      key: {
+        text: "Jurisdiction"
+      },
+      value: {
+        text: appData.entity.public_register_jurisdiction
```

Comment:
This could fall under the bug K1 have for country formed in. Correct here I think but showing wrong results.